lib/index.js

```diff
@@ -297,8 +324,8 @@ module.exports = (function() {
             // Add newly inserted rows
             if(newTableRows.length > 0) {
                 //Move images for each subsitute array if option is active
-                if(self.option["moveImages"]){
-                    var rels = self.loadSheetRels(sheet.filename);
+                if(self.option["moveImages"] && rels){
+                    //var rels = self.loadSheetRels(sheet.filename);
                     if (rels != null && drawing == null){
```

Comment:
`rels != null` condition not nescessary, and as such later conditions can be simplified I belive.

Reply:
Ok (commit a1fb38e). I added a comment. What do you think about call the function self.loadDrawing() at the begining of substitute() function, just before sheetData.findall("row").forEach(function(row), like loadSheetRels() function.
this will create the drawing file all the file (like sheet rels file with loadSheetRels()).